dev/user.clj

```diff
@@ -17,13 +17,13 @@
 (alias 'pass (create-ns 'juxt.pass.alpha))
 (alias 'site (create-ns 'juxt.site.alpha))
 
-(defn start []
+(defn start [& [config-file-in]]
```

Comment:
Feels like if we're going to add command line options, we should go for broke and bring in tools.cli

Reply:
latest Clojure CLI tools already support a more lightweight solution via the -X option, so I would rather start with that, instead of pulling in tools.cli

Reply:
Wasn't aware of -X, thanks